### PR TITLE
test: fix binlog 999999 test

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What problem does this PR solve?

Issue Number: ref #4159

### What is changed and how it works?
 copy db file and add reset master instead.

### Release note 

```release-note
None
```

### CODE REVIEW

File: dm/tests/incremental_mode/data/db1.prepare.sql [hunk 1]
```diff
@@ -1,5 +1,4 @@
 drop database if exists `incremental_mode`;
-reset master;
```
Comment (D3Hunter) [inline dm/tests/incremental_mode/data/db1.prepare.sql:2]: please add a clear message in this files, so we won't add reset master later mistakenly.

also please add a clear comments in run.sh that it's used in chaos test 99999, something like:
```
############################
#            xxxxxxxxxxxxxxxxxxx
############################
```

**Review by lance6716 — APPROVED** [review]
please address comment 👆

File: dm/tests/incremental_mode/data/db1.prepare.sql [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* Should not add reset master in this file */
```
Comment (D3Hunter) [inline dm/tests/incremental_mode/data/db1.prepare.sql:1]: maybe explain why should not do it

**Review by D3Hunter — APPROVED** [review]

**Comment by lance6716** [comment]
```
[2022-05-13T07:10:55.029Z] [2022/05/13 15:00:52.970 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:01:23.973 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:01:54.975 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:02:25.977 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:02:56.980 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:03:27.981 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:03:58.984 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:04:29.985 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:05:00.989 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:05:32.166 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:06:04.320 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:06:36.480 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:07:08.640 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:07:40.819 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:08:12.985 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:08:45.161 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:09:17.332 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:09:49.513 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:10:21.692 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
```

```
[2022-05-13T07:10:54.427Z] command: query-status test "unit": "Sync" count: 1 != expected: 2, failed the 9-th time, will retry again
[2022-05-13T07:10:54.427Z] {
[2022-05-13T07:10:54.427Z]     "result": true,
[2022-05-13T07:10:54.427Z]     "msg": "",
[2022-05-13T07:10:54.427Z]     "sources": [
[2022-05-13T07:10:54.427Z]         {
[2022-05-13T07:10:54.427Z]             "result": false,
[2022-05-13T07:10:54.427Z]             "msg": "[code=38008:class=dm-master:scope=internal:level=high], Message: grpc request error, RawCause: rpc error: code = DeadlineExceeded desc = context deadline exceeded",
[2022-05-13T07:10:54.427Z]             "sourceStatus": {
[2022-05-13T07:10:54.427Z]                 "source": "mysql-replica-01",
[2022-05-13T07:10:54.427Z]                 "worker": "",
[2022-05-13T07:10:54.427Z]                 "result": null,
[2022-05-13T07:10:54.427Z]                 "relayStatus": null
[2022-05-13T07:10:54.427Z]             },
[2022-05-13T07:10:54.427Z]             "subTaskStatus": [
[2022-05-13T07:10:54.427Z]             ]
[2022-05-13T07:10:54.427Z]         },
```

maybe we should revert https://github.com/pingcap/tiflow/pull/5350 ? seems there's deadlock but I can't reproduce

**Comment by lance6716** [comment]
maybe the deadlock is not introduced by #5350.

```
[2022-05-13T07:10:55.029Z] [2022/05/13 15:00:21.979 +08:00] [INFO] [source_worker.go:820] ["start to create subtask in operateSubTaskStage"] [sourceID=mysql-replica-01] [task=test]
```
lack `subtask created`


```
[2022-05-13T07:10:55.029Z] + cat /tmp/dm_test/shardddl2/worker1/log/dm-worker.log
[2022-05-13T07:10:55.029Z] [2022/05/13 15:00:21.967 +08:00] [INFO] [source_worker.go:780] ["receive subtask stage change"] [stage="{\"expect\":2,\"source\":\"mysql-replica-01\",\"task\":\"test\"}"] ["is deleted"=false]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:00:21.969 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:00:21.979 +08:00] [INFO] [source_worker.go:810] [operateSubTaskStage] [sourceID=mysql-replica-01] [task=test] [stage="{\"expect\":2,\"source\":\"mysql-replica-01\",\"task\":\"test\"}"]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:00:21.979 +08:00] [INFO] [source_worker.go:820] ["start to create subtask in operateSubTaskStage"] [sourceID=mysql-replica-01] [task=test]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:00:52.970 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:01:23.973 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:01:54.975 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:02:25.977 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.029Z] [2022/05/13 15:02:56.980 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:03:27.981 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:03:58.984 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:04:29.985 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:05:00.989 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:05:32.166 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:06:04.320 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:06:36.480 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:07:08.640 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:07:40.819 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:08:12.985 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:08:45.161 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:09:17.332 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:09:49.513 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] [2022/05/13 15:10:21.692 +08:00] [INFO] [server.go:744] [request=QueryStatus] [payload="name:\"test\" "]
[2022-05-13T07:10:55.030Z] + echo ____________________________________
```

**Pull request merged**